src/List.test.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+
+import List from './List';
+
+describe('List', () => {
+  const handleClick = jest.fn();
+
+  function renderList({ tasks, handleClickDelete }) {
+    return render((
+      <List
+        tasks={tasks}
+        onClickDelete={handleClickDelete}
+      />
+    ));
+  }
```

Comment:
```suggestion
  const handleClickDelete = jest.fn();

  function renderList(tasks) {
    return render((
      <List
        tasks={tasks}
        onClickDelete={handleClickDelete}
      />
    ));
  }
```

굳이 호출시점에 핸들러 함수를 넘길 필요 없이 이렇게 처리하면 인자 하나를 없앨 수 있을 것 같아요 ☺️